types/node/crypto.d.ts

```diff
@@ -2475,6 +2481,10 @@ declare module "crypto" {
          * Name of the curve to use
          */
         namedCurve: string;
+        /**
+         * Must be `'named'` or `'explicit'`. Default: `'named'`.
+         */
+        paramEncoding?: "explicit" | "named";
```

Comment:
```suggestion
        paramEncoding?: "explicit" | "named" | undefined;
```

for `--exactOptionalPropertyTypes`